.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: 'v2.4.0'
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-merge-conflict
+    -   id: flake8
```

Comment:
Elsewhere internally at Quantopian we're using:

```
  - repo: https://gitlab.com/pycqa/flake8
    rev: 3.7.7
    hooks:
    - id: flake8
```

which has the benefit that it allows us to pin a specific version of flake8. I'm not sure if there's a way to do the using the flake8 plugin supplied by `pre-commit`.

Reply:
Yeah, I actually used that way before too, but since it was added to the default list of default hooks I switched for simplicity.

But sure, I can revert o that way too.